lib/system-nixpkgs-map.nix

```diff
@@ -84,6 +85,17 @@ with pkgs;
   vulkan = [ vulkan-loader ];
   sodium = [ libsodium ];
   gfortran = [ gfortran.cc.lib ];
+  png = [ libpng ];
+  jpeg = [ libjpeg ];
+  m = []; # Included with ghc
+  freenect_sync = [ freenect ];
+  FLAC = [ flac ];
+  mp3lame = [ lame ];
+  tag_c = [ taglib ];
+  jwt = [ libjwt ];
+  GeoIP = [ geoip ];
+  pulse-simple = [ libpulseaudio ];
+  oath = [ liboauth ];
```

Comment:
to be annoying: should we alphabetically sort this set to reduce the risk of duplicates?